SEC/Equities/EquityInstruments.rdf

```diff
@@ -273,7 +287,7 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasSchedule"/>
-				<owl:onClass rdf:resource="&fibo-sec-eq-eq;DividendOrShareCouponSchedule"/>
+				<owl:onClass rdf:resource="&fibo-sec-eq-eq;DividendSchedule"/>
```

Comment:
They are both optional restrictions.  The declaration is always a dollar amount per share, not a percentage, and there may be a schedule or not, for example, for quarterly payments at that dollar per share amount for some window of time before the board elects to change the amount.  They may have internally agreed on a percentage at the board level, but what is published is typically an amount not a percentage.

Reply:
Per discussion in the FCT, and confirming with Citi, a given dividend, especially for a common share, is specified as an amount, and if they need to, they compute a percentage.  But boards of directors typically approve a dollar amount.  The optional schedule is more commonly associated with a preferred dividend, though some boards approve quarterly dividends for the coming year for common shares.

Reply:
For preferred shares, it might be stated as a percentage of the par value, but for common stock it is a dollar amount (confirmed with Citi).  But if you state it one way, you can derive the other.

Reply:
As a consequence of the discussion, we've made the dividend amount (monetary amount) required for ordinary dividends, but have not added that restriction to preferred dividends, where it may be expressed as a percentage or as an amount.

Reply:
(Really need an example)
What is the Commitment then? If in 2019 a Board declared quarterly dividends of $1, $2, none and $1 then are there 4 Dividend instances? Are they created only once a Board has made its determination that quarter? Is there a Schedule at all? How do we capture generally that they are committed to announce a Dividend quarterly (even if it is zero)?

Reply:
Raised issue